src/osx/core/colour.cpp

```diff
@@ -13,127 +13,244 @@
 #include "wx/colour.h"
 
 #ifndef WX_PRECOMP
-    #include "wx/gdicmn.h"
+#include "wx/gdicmn.h"
```

Comment:
Again, I'm sorry for nitpicking, but I actually think indenting the preprocessor directives is nice and I don't understand why does this need to be changed :-(

Reply:
I haven't found out how to let clang-format indent the pragma that way, would `PPDIS_AfterHash` be an option ?